tests/envs/mujoco/test_mujoco_rendering.py

```diff
@@ -94,6 +99,27 @@ def test_camera_id(render_mode: str):
     assert env_a.mujoco_renderer.camera_id == env_b.mujoco_renderer.camera_id
     assert env_a.mujoco_renderer.camera_id != env_c.mujoco_renderer.camera_id
 
-    if render_mode != "human":
+    if render_mode == "rgbd_tuple":
+        rgb_a, depth_a = env_a.render()
+        rgb_b, depth_b = env_b.render()
+        rgb_c, depth_c = env_c.render()
+        assert (rgb_a == rgb_b).all()
+        assert (depth_a == depth_b).all()
+        assert (rgb_a != rgb_c).any()
+        assert (depth_a != depth_c).any()
+
+    elif render_mode != "human":
         assert (env_a.render() == env_b.render()).all()
         assert (env_a.render() != env_c.render()).any()
+
+
+def test_rgbd_tuple():
+    """Assert that rgbd_tuple is the proper combination of rgb and depth images as tuple"""
+    env = gymnasium.make("Ant-v5", camera_id=0, render_mode="rgbd_tuple").unwrapped
```

Comment:
render once with `rgb_array` and `depth_array` and assert that the resulting images are the same